06_neighborhoods/Transportation/transit_trips_cost_city.R

```diff
@@ -0,0 +1,283 @@
+
+
+###################################################################
+
+# Transit Metrics: Transportation Costs & Trips - CITY LEVEL
```

Comment:
This script does not load any packages. At a minimum I believe it needs at least the tidyverse package to run.